<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VAITP</class>
 <widget class="QMainWindow" name="VAITP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1435</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VAITP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo/icon_48.png</normaloff>:/logo/icon_48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>851</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>VAITP - Vulnerability Attack and Injection Tool for Python v0.1 Beta</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1421</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Local</string>
     </attribute>
     <widget class="QLineEdit" name="txt_py_src">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="bt_load_py_src">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>19</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a Python file:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_injectionPoints">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>341</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vulnerabilities:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_vulns">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>341</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>271</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Injection points:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_scan_py">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Scan File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_inject_vuln">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>250</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inject Vulnerability</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_restore_pys">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Restore un-injected files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_attack">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>250</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Single Attack</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>80</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Attack payloads:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txt_output_sh1">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>1041</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>300</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_export_output_sh1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>480</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Export output</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_payload">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>100</y>
        <width>341</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_clearAll">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>480</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_injectedFiles">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>341</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Injected files:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_auto_daisyChain">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Daisy-Chain Attack</string>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>80</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Working attacks:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lst_workingAttacks">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>100</y>
        <width>341</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Remote</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1435</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>VAITP</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
